web-client/src/presenter/computeds/practitionerDocumentationFormHelper.test.js

```diff
@@ -0,0 +1,68 @@
+import { PRACTITIONER_DOCUMENT_TYPES_MAP } from '../../../../shared/src/business/entities/EntityConstants';
+import { applicationContext } from '../../applicationContext';
+import { practitionerDocumentationFormHelper as practitionerDocumentationFormHelperComputed } from './practitionerDocumentationFormHelper';
+import { runCompute } from 'cerebral/test';
+import { withAppContextDecorator } from '../../withAppContext';
+
+describe('practitionerDocumentationFormHelper', () => {
+  const practitionerDocumentationFormHelper = withAppContextDecorator(
+    practitionerDocumentationFormHelperComputed,
+    {
+      ...applicationContext,
+    },
+  );
+
+  describe('isCertificateOfGoodStanding', () => {
+    it('should return true for isCertificateOfGoodStanding if documentationCategory equals "Certificate of Good Standing"', () => {
+      const { isCertificateOfGoodStanding } = runCompute(
+        practitionerDocumentationFormHelper,
+        {
+          state: {
+            form: {
+              categoryType:
+                PRACTITIONER_DOCUMENT_TYPES_MAP.CERTIFICATE_OF_GOOD_STANDING,
+            },
+            permissions: {
+              UPLOAD_PRACTITIONER_DOCUMENT: true,
+            },
+          },
+        },
+      );
+      expect(isCertificateOfGoodStanding).toBe(true);
+    });
+
+    it('should return false for isCertificateOfGoodStanding if documentationCategory equals "Certificate of Good Standing"', () => {
```

Comment:
i think this it statement also needs to be amended.

Reply:
rephrased to 'should return false for isCertificateOfGoodStanding if categoryType is undefined'